pyproject.toml

```diff
@@ -49,6 +49,7 @@ python = "^3.7"
 # TODO: Remove this dependency when Python 3.7 support is removed
 #       https://github.com/phylum-dev/phylum-ci/issues/18
 importlib-metadata = {version = "^4.11.3", python = "<3.8"}
+PyYAML = "5.3.1"
```

Comment:
This is a known bad package added temporarily to confirm the PR analyze check is working. It will be taken out before this review is merged.